higgstautau/lephad/branches.py

```diff
@@ -0,0 +1,190 @@
+# branches to ignore in the input tree and not include in the output tree
```

Comment:
@vincecr0ft we could probably only have one of those files for the three channels